DOCS/man/options.rst

```diff
@@ -1546,14 +1546,27 @@ Video
 
 ``--video-pan-x=<value>``, ``--video-pan-y=<value>``
     Moves the displayed video rectangle by the given value in the X or Y
-    direction. The unit is in fractions of the size of the destination rectangle.
+    direction. The unit is in fractions of the size of the scaled video (the
+    full size, even if parts of the video are not visible due to panscan or
+    other options).
 
-    For example, displaying a video fullscreen on a 1920x1080 screen with
-    ``--video-pan-x=-0.1`` would move the video 192 pixels to the left and
-    ``--video-pan-y=-0.1`` would move the video 108 pixels up.
+    For example, displaying a video scaled to 1920x1080 due to video-zoom or
+    other options on a 1280x720 mpv window, with ``--video-pan-x=-0.1`` would
+    move the video 192 pixels to the left and ``--video-pan-y=-0.1`` would move
+    the video 108 pixels up.
 
     This option is disabled if the ``--no-keepaspect`` option is used.
 
+``--output-pan-x=<value>``, ``--output-pan-y=<value>``
+    Same as ``--video-pan-x`` and ``--video-pan-y`` except unit of the value is
+    in fractions of the size of the destination rectangle.
+
+    For example, displaying a video fullscreen on a 1920x1080 screen with
+    ``--output-pan-x=-0.1`` would move the video 192 pixels to the left and
+    ``--output-pan-y=-0.1`` would move the video 108 pixels up.
```

Comment:
Now that we have both options, I think the example to be useful should show the difference between them.